README.md

```diff
@@ -481,6 +481,14 @@ yargs.command('get', 'make a get HTTP request', {
   .argv
 ```
 
+Note that commands will not automatically inherit configuration _or_ options
+of their parent context. This means you'll have to re-apply configuration
+if necessary, and make options global manually using the [global](#global) method.
+
+Additionally, the [`help`](#helpoption) and [`version`](#versionoption)
```

Comment:
Looks like we might want to add explicit anchors for `.help()` and `.version()` in the README since these links don't seem to work in GitHub or npmjs.com.